ledger/src/blockstore_db.rs

```diff
@@ -303,13 +302,29 @@ impl Rocks {
                         db_options.set_max_open_files(-1);
 
                         Rocks(
-                            DB::open_cf_as_secondary(&db_options, path, &secondary_path, names)?,
+                            DB::open_cf_as_secondary(
+                                &db_options,
+                                path,
+                                &secondary_path,
+                                cf_names.clone(),
+                            )?,
                             ActualAccessType::Secondary,
                         )
                     }
                 }
             }
         };
+        for cf_name in cf_names {
+            db.0.set_options_cf(
```

Comment:
at least, I confirmed this  is working as intended according to `rocksdb/OPTIONS` file.